src/main/resources/public/src/views/Validation.vue

```diff
@@ -1,43 +1,68 @@
 <template>
     <div class="container">
         <h2>Access validation</h2>
-        <data-access-validation></data-access-validation>
-        <router-link to="/">
-            <input class="validation-buttons" type="button" @click="validAccess" value="Valid" v-bind:disabled="disabled">
-        </router-link>
-        <router-link to="/">
-            <input class="validation-buttons" type="button" @click="refuseAccess" value="Refuse">
-        </router-link>
+        <p>{{ dataRequest.model }}</p>
+        <div class="form-check">
+            <input id="agreeAllData" class="form-check-input" type="checkbox" :value="agreeAllData" @change="handleAgreeAllData">
+            <label for="agreeAllData" class="form-check-label">
+                Authorize access to all data of "{{ dataRequest.model }}" model
```

Comment:
dates are hard-localized in FR but messages are in EN. A choice has to be made (and FR for this very first iteration is a sensible choice).